docs/pages/Cardinal/_meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "CoreConcepts": {
```

Comment:
I don't think "Core Concepts" is apt here. Since this part of the docs is more of an API reference, we should call it "API Reference instead